.flowconfig

```diff
@@ -1,6 +1,6 @@
 [ignore]
 ; We fork some components by platform
-.*/*[.]android.js
+<PROJECT_ROOT>/node_modules/.*/*[.]android.js
```

Comment:
This should only be in place for dependencies. Maybe we should scope it to `react-native` only...🤔

Reply:
Yeah let's scope it for RN and see what happens

Reply:
Fixed